sdk/nodejs/appservice/zMixins_http.ts

```diff
@@ -122,19 +122,21 @@ export class HttpEventSubscription extends mod.EventSubscription<mod.Context<Htt
         const { resourceGroupName, location } = mod.getResourceGroupNameAndLocation(args, undefined);
 
         const bindings: HttpBindingDefinition[] = [{
-                    authLevel: "anonymous",
-                    type: "httpTrigger",
-                    direction: "in",
-                    name: "req",
-                    route: args.route,
-                    methods: args.methods,
-                }, {
-                    type: "http",
-                    direction: "out",
-                    name: "$return",
-                }];
-
-        super("azure:appservice:HttpEventSubscription", name, bindings, {
+            authLevel: "anonymous",
+            type: "httpTrigger",
+            direction: "in",
+            name: "req",
+            route: args.route,
+            methods: args.methods,
+        }, {
+            type: "http",
+            direction: "out",
+            name: "$return",
+        }];
+
+        args.bindings = mod.mergeBindings(bindings, args.bindings);

```

Comment:
i'd prefer we not mutate values flowing in.  so instead, just write this all as:

```ts
        super("azure:appservice:HttpEventSubscription", name, {
            ...args,
            bindings:  mod.mergeBindings(bindings, args.bindings),
```

Reply:
fixed in d783c138f08bff57325be4e7fb98e3a6933d5b8b